test/integ/routers/alpha-router/alpha-router.integration.test.ts

```diff
@@ -859,220 +1389,7 @@ describe('alpha router integration', () => {
       });
     });
   }
-
-  describe('Mixed routes', () => {
```

Comment:
Why are these tests deleted? These are in main and should be merged into your feature